packages/office-ui-fabric-react/src/components/Callout/Callout.types.ts

```diff
@@ -138,7 +139,7 @@ export interface ICalloutProps extends React.Props<Callout | CalloutContent> {
   /**
    * Optional callback that is called once the callout has been correctly positioned.
    */
-  onPositioned?: () => void;
+  onPositioned?: (positions?: IPositionDirectionalHintData) => void;
```

Comment:
What is the positions param? (add comments)

Reply:
Done